test/unit/test_blas_tile_extensions.cpp

```diff
@@ -0,0 +1,91 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2023, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#include <dlaf/blas/tile.h>
+
+#include "test_blas_tile/test_add.h"
+#include "test_blas_tile/test_scal.h"
+
+#include <gtest/gtest.h>
+
+using namespace dlaf;
+using namespace dlaf::test;
+using namespace testing;
+template <class T, Device D>
+class TileOperationsTest : public ::testing::Test {};
+
+template <class T>
+using TileOperationsExtensionsTestMC = TileOperationsTest<T, Device::CPU>;
+
+TYPED_TEST_SUITE(TileOperationsExtensionsTestMC, MatrixElementTypes);
+
+#ifdef DLAF_WITH_GPU
+template <class T>
+using TileOperationsExtensionsTestGPU = TileOperationsTest<T, Device::GPU>;
+
+TYPED_TEST_SUITE(TileOperationsExtensionsTestGPU, MatrixElementTypes);
+#endif
+
+// Tuple elements:  m, n, k, extra_lda, extra_ldb, extra_ldc
+std::vector<std::tuple<SizeType, SizeType, SizeType, SizeType, SizeType, SizeType>> gemm_sizes = {
```

Comment:
Why is this called `gemm_sizes`?
You should create `scal_sizes` and `add_sizes` with only the parameters needed by the corresponding test.